solutions/swb-reference/src/SWBStack.ts

```diff
@@ -210,29 +219,6 @@ export class SWBStack extends Stack {
     });
   }
 
-  private _createEncryptionKey(): Key {
-    const mainKeyPolicy = new PolicyDocument({
-      statements: [
-        new PolicyStatement({
-          actions: ['kms:*'],
-          principals: [new AccountPrincipal(this.account)],
-          resources: ['*'],
-          sid: 'main-key-share-statement'
-        })
-      ]
-    });
-
-    const key = new Key(this, 'mainAccountKey', {
-      enableKeyRotation: true,
```

Comment:
If this key already had key rotation enabled, were you able to identify if there were any other keys in usage by SWB that did not have key rotation enabled? Just want to make sure we've addressed the original app sec finding that led to this ticket.

Reply:
verified. s3 artifact, s3 datasets and ddb table were the resources the ticket referring to